src/main/java/command/Commands.java

```diff
@@ -0,0 +1,351 @@
+package command;
+
+
+import task.ListOfTask;
+import dukeExceptions.DukeException;
+
+import java.time.LocalDateTime;
+
+public class Commands {
+
+    /**
+     * List of both primary and secondary commands.
+     */
+    public enum COMMANDS {BYE, LIST, TODO, DEADLINE, EVENT, MARK, UNMARK, DELETE, BY, FROM, TO, SORT, FIND, UNKNOWN}
+
+    private COMMANDS state;
+    private String name;
+    private int index;
+    private LocalDateTime dateTime;
+
+    /**
+     * Construct a Commands object with only the primary command.

```

Comment:
Perhaps verbs used in JavaDocs should be singular tense (e.g. Constructs instead of Construct).

Reply:
Roger that I'll change